dtfs-central-api/test-helpers/test-data/tfm-facility.ts

```diff
@@ -4,16 +4,13 @@ import { generateMockPortalUserAuditDatabaseRecord } from '@ukef/dtfs2-common/ch
 import { aFacility } from './facility';
 import { KeyingSheetCalculationFacilityValues } from '../../src/types/tfm/tfm-facility';
 
-const facility = aFacility();
-const { dayCountBasis, interestPercentage, coverPercentage, value } = facility;
-
 export const aTfmFacility = (): TfmFacility => {
   const tfmFacilityId = new ObjectId();
 
   return {
     _id: tfmFacilityId,
     facilitySnapshot: {
-      ...facility,
```

Comment:
This was not instantiating the facility within the `aTfmFacility` function. An object on this facility could be modified & it would affect all instances of `aTfmFacility`